src/Proto3/Suite/DotProto/Generate.hs

```diff
@@ -880,40 +943,58 @@ fromJSONPBMessageInstD _ctxt parentIdent msgIdent messageParts = do
   msgName    <- nestedTypeName parentIdent =<< dpIdentUnqualName msgIdent
   qualFields <- getQualifiedFields msgName messageParts
 
+  let lambdaPVar = patVar "obj"
+  let lambdaVar  = HsVar (unqual_ "obj")
   -- E.g., for message Something{ oneof name_or_id { string name = _; int32 someid = _; } }:
```

Comment:
Is this example still accurate?

Reply:
This generated code is now nested within more code.

Reply:
Moved comment.